apcd-cms/src/apps/view_users/views.py

```diff
@@ -36,93 +48,72 @@ def _edit_user(form):
                 logger.debug(print("success"))
                 template = loader.get_template('view_user_edit_success.html')
             return template
-        
+
         template = _edit_user(form)
         return HttpResponse(template.render({}, request))
-    def get(self, request, *args, **kwargs):
-        user_content = get_users()
 
-
-        context = self.get_context_data(user_content, *args,**kwargs)
-        template = loader.get_template(self.template_name)
-        return HttpResponse(template.render(context, request))
-        ##END FORM FUNCTION
-
-    user_content = get_users()
+    # def get(self, request, *args, **kwargs):
+    #     user_content = get_users()
+    #     # print(f"Get API users: {get_api_users()}")
+    #
+    #     context = self.get_context_data(user_content, *args, **kwargs)
+    #     template = loader.get_template(self.template_name)
+    #     return HttpResponse(template.render(context, request))
+    #     ##END FORM FUNCTION
+    #
+    # user_content = get_users()
 
     def dispatch(self, request, *args, **kwargs):
         if not request.user.is_authenticated or not is_apcd_admin(request.user):
-           return HttpResponseRedirect('/')
+            return HttpResponseRedirect('/')
         return super(ViewUsersTable, self).dispatch(request, *args, **kwargs)
 
-    def get_context_data(self, user_content=user_content, *args, **kwargs):
+    def get_context_data(self, *args, **kwargs):
         context = super(ViewUsersTable, self).get_context_data(*args, **kwargs)
 
-        def _set_user(usr):
-            return {
-                    'role_id': usr[0],
-                    'user_id': usr[1],
-                    'user_email': usr[2],
-                    'user_name': usr[3],
-                    'org_name': usr[4],
-                    'created_at': usr[5],
-                    'updated_at': usr[6],
-                    'notes': usr[7],
-                    'status': 'Active' if usr[8] else 'Inactive',
-                    'user_number': usr[9],
-                    'role_name': usr[10],
-                    'org_name_no_parens': usr[4].replace("(", "").replace(")", ""),  # just for filtering purposes
-                    'active': usr[8],
-                }
-
-        context['header'] = ['User ID', 'Name', 'Organization', 'Role', 'Status', 'User Number', 'See More']
+        context['header'] = ['User ID', 'Name', 'Organization', 'Role', 'Active', 'User Number', 'See More']
         context['status_options'] = ['All', 'Active', 'Inactive']
-        context['filter_options'] = ['All']
-        context['role_options'] = ['SUBMITTER_USER', 'SUBMITTER_ADMIN','APCD_ADMIN']
-
-        context['status'] = ['Active', 'Inactive']
+        context['role_options'] = ['SUBMITTER_USER', 'SUBMITTER_ADMIN', 'APCD_ADMIN']
+        context['status'] = ['True', 'False']
+        context['filter_options'] = ['All'] + get_api_users_org_list()
```

Comment:
This is throwing an error 'can only concatenate list (not "dict") to list'.

You could try something like 
` org_list = get_api_users_org_list()
        for i in org_list:
            context['filter_options'].append(i)`